ansible/roles/ocp4-workload-cnvlab/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+become_override: False
+ocp_username: kubeadmin
+silent: False
+
+# If your workload needs customization options provide variables to be used.
+# Variable names must be prefixed by the role name "<role name>_<variable>".
+# Therefore because this example workload is called "ocp4_workload_example" the variables
+# should be named "ocp4_workload_example_variable".
+#
+# You can override the defaults as parameters to the Ansible run that runs
+# your workload.
+
+# Documentation for variable 1 (what does it do?)
+ocp4_workload_cnvlab_variable_1: "value 1"
```

Comment:
Do you even need those? If not then please remove the variables.

Reply:
Thanks - they're gone!